src/test/resources/IdTest/geoLocationIdsOnly.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
No Query.java usage? :(

Reply:
I actually wrote it that way originally, but then saw our previous commit to the file and changed it.
Which way do you want to go?

    @Test
    public void testGeoLocationIdsOnlyWithCorrectIdFieldName() {
        JsonPath postBody = getJson("geoLocationIdsOnly.json");
        postQuery(postBody)
            .body("metadata.idField", is("ID"))
            .body("metadata.fields[0].name", is("ID"))
            .body("features[0].ID", is(15))
            .body("features[0].properties.ID", is(15))
        ;
    }

Reply:
Ok, I like it now.